<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QDialog" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>330</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>【修改密码】</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户代码：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditM1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>原来密码：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPO">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPN">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>新的密码：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPR">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>重复密码：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户姓名：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditM2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>【新增用户】</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户代码：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditA1">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户姓名：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditA2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditAP1">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置密码：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置权限：</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxP">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>重复密码：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditAP2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonM">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>确认修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonE">
        <property name="text">
         <string>取消退出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>确认增加</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
